### ISSUE TITLE
[Accessibility] [Web] [Profile Page] - "Edit" button lacks translated label

### ISSUE THREAD

**Issue by ceciliabelem** [cambiatus/frontend#970]
**Type:** labeling  
**Severity:** High  

**Describe the bug**  
The "edit" button does not have a descriptive accessible label. It should be announced as "edit" in English, but in Portuguese, it should be "Editar" as user's language.

**To Reproduce**  
1. Go to 'https://muda.cambiatus.io/profile/user_profile'  
2. Focus on the "edit" button with NVDA  
3. Notice the lack of a descriptive label

**Expected behavior**  
The button should be announced as "edit profile".

**Screenshots**  

<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/4172a170-566f-4a68-9fba-e6136a70d8b7" />

**Cambiatus Env you are using**  
https://muda.cambiatus.io/profile/user_profile

**Desktop (please complete the following information):**  
- OS: Windows 10  
- Browser: Chrome  
- Version: 139.0.7258.139

**Smartphone (please complete the following information):**  
- Device: N/A  
- OS: N/A  
- Browser: N/A  
- Version: N/A

**Additional context**  
Tested with NVDA 2025.2.
